pep-0706.rst

```diff
@@ -13,7 +13,7 @@ Post-History: `25-Jan-2023 <https://discuss.python.org/t/23149>`__,
 Abstract
 ========
 
-The extraction methods in ``tarfile`` gain a ``filter`` argument,
+The extraction methods in :external+py3.11:mod:`tarfile` gain a ``filter`` argument,
```

Comment:
```suggestion
The extraction methods in :mod:`py3.11:tarfile` gain a ``filter`` argument,
```

As you mentioned, the full `:external:` form is quite long and ugly; I would suggest just using the shorter and much more widely used (AFAIK) prefix form instead, but ultimately up to you of course.

Reply:
I asked on the docs discord. I'd rather not rely on an undocumented feature, too often those turn out to be implementation accidents or have weird edge cases that change a few years later.

Reply:
Based on @AA-Turner 's response there, it turns out both are fully supported and the shorter form more widely used, but of course up to your preference if we want to go back through it at some point and change it.